assets/training/model_management/components/import_model/spec.yaml

```diff
@@ -38,6 +38,11 @@ inputs:
     type: string
     description: A valid model id for the model source selected. For example you can specify `bert-base-uncased` for importing HuggingFace bert base uncased model. Please specify the complete URL if **GIT** or **AzureBlob** is selected in `model_source`
 
+  hf_authentication_token:
```

Comment:
can this be generically defined as token? Underlying download will figure out what this means and apply as necessary?